src/test/java/integration/EventsTest.java

```diff
@@ -882,8 +884,9 @@ public void given_multibranchWithSourcesAndTwoBranches_when_indexing_then_deadBr
             prj.scheduleBuild2(0).getFuture().get();
             r.waitUntilNoActivity();
             assertThat("Indexing applies dead branch cleanup",
-                    prj.getItems(), containsInAnyOrder(master));
```

Comment:
Reverting this as `containsInAnyOrder(master)` checks that the contents are `master` and nothing else so the `allOf` is redundant